numpy/lib/nanfunctions.py

```diff
@@ -306,7 +306,7 @@ def nanmax(a, axis=None, out=None, keepdims=np._NoValue):
     a : array_like
         Array containing numbers whose maximum is desired. If `a` is not an
         array, a conversion is attempted.
-    axis : int, optional
+    axis : {int, tuple of int, None}, optional
         Axis along which the maximum is computed. The default is to compute
```

Comment:
These should all be updated to say `Axis or axes`, which would match the docs for `sum` and others